staging/src/k8s.io/client-go/copy.sh

```diff
@@ -18,17 +18,36 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+VERIFYONLY=false
+while getopts ":v" opt; do
+  case $opt in
+    v)
+      VERIFYONLY=true
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      ;;
+  esac
+done
```

Comment:
You could mark `VERIFYONLY` as read-only after the completion of the while loop, i.e.,

`readonly VERIFYONLY`